packages/dso-toolkit/src/components/alert/alert.mixins.scss

```diff
@@ -53,23 +53,22 @@
 }
 
 @mixin status-success() {
-  @include set-colors.apply(alert-variables.$success-bg, alert-variables.$success-border);
+  @include set-colors.apply(alert-variables.$success-bg);
```

Comment:
Goed dat die border color is verdwenen. Alert heeft helemaal geen border rondom, de variables zijn een erfenis van Bootstrap.

Mogelijk andere naam bedenken voor `alert-small-border-color` (edge?), nadat alles wat met "het randje om het Alert" te maken heeft weg is gehaald (Sass variables etc).

Reply:
Gedaan